assets/src/block-visibility/check.js

```diff
@@ -48,5 +48,6 @@ const getBlacklist = () => {
 		 * @link https://github.com/gocodebox/lifterlms-blocks/issues/41
 		 */
 		'core/freeform',
+		'llms/course-progress'
```

Comment:
Instead of blacklisting the block you can define that the block *does not support* visibility during block registration.

This blacklist was added to allow us to blacklist blocks that we don't have control of, whereas if we have a block we do have control of that we don't want to support visibility we should define it doesn't support it.

Here's an example in the lesson progression block (pending pr): https://github.com/gocodebox/lifterlms-blocks/blob/236ecbd4b25b7257cd1730149812223b17a4563f/assets/src/blocks/lesson-progression/index.js#L48-L50

Reply:
ah yeah better

Reply:
I read "better" as "butter" -- I might be hungry...